src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const PRICE_PER_DAY = 40 * days;
```

Comment:
```suggestion
  const pricePerDay = 40 * days;
```
it's not the constant 'cause depends on the `days` function argument